src/main/getDiForUnitTesting.ts

```diff
@@ -55,6 +72,26 @@ export const getDiForUnitTesting = (
 
   di.preventSideEffects();
 
+  if (overrideHotbarStore) {
```

Comment:
Ifs are redundant with `di.unoverride()`. Also usage of unoverride is a probable sign of unit test with side-effects. Proper fix is to get rid of the side-effect in unit test.